src/components/Pill/Pill.stories.tsx

```diff
@@ -67,11 +52,17 @@ Removable.args = {
   size: 'sm',
 };
 
-export const Clickable = PillTemplate.bind({});
-Clickable.args = {
-  ...Playground.args,
-  onClick: action('OnRemove'),
-};
+export const Clickable: Story<PillProps> = () => (
+  <Inline gap="md">
+    {Object.values(PillColorsEnums).map((color) => (
+      <Pill
+        color={color}
+        label={capitalize(color)}
+        onClick={action('OnRemove')}
+      />
+    ))}
+  </Inline>
+);
 
 export const ClickableAndDeletable = PillTemplate.bind({});
 ClickableAndDeletable.args = {
```

Comment:
You removed `args` definition for `Clickable` story which this story uses. Due to this the pill in this story is no longer clickable.

You can revert the changes for the `Clickable` story and just reuse `args` property for all instances like:
```jsx
export const Clickable: Story<PillProps> = (args) => (
  <Inline gap="md">
    {Object.values(PillColorsEnums).map((color) => (
      <Pill
        key={color}
        {...args}
        color={color}
      />
    ))}
  </Inline>
);

Clickable.args = {
  ...Playground.args,
  onClick: action('OnRemove'),
};